appeals/web/src/server/lib/mappers/components/text.js

```diff
@@ -53,7 +55,7 @@ export function textDisplayField({
 	const actions = [];
 	if (userHasEditPermission) {
```

Comment:
I think this should be named editable instead, as it won't always be because of permission. We could use it to hide actions based on status

Reply:
This is true, and we do use it for that in places. I will do that rename separately as it will be lots of little changes.